.github/workflows/tests.yml

```diff
@@ -21,6 +21,24 @@ jobs:
     - name: Acquire sources
       uses: actions/checkout@v3
 
+    # https://github.com/docker-practice/actions-setup-docker
+    # - name: Install Docker
+    #   if: runner.os == 'Linux'
+    #   uses: docker-practice/actions-setup-docker@master
+
+    - name: Display Docker version
+      if: runner.os == 'Linux'
+      run: |
+        docker version
+
+    - name: Install prerequisites
+      if: runner.os == 'Linux'
+      run: |
+        # Workaround against `404 Client Error: Not Found for url: http+docker://localhost/v1.23/containers/create`.
+        # https://github.com/jpmens/mqttwarn/pull/589#issuecomment-1249680740
+        # https://github.com/docker/docker-py/issues/2101
+        docker pull eclipse-mosquitto:2.0.15
```

Comment:
Note to self: Re-evaluate this, eventually fix differently.

Reply:
The workaround is now part of `pytest-mqtt`, see [1].

[1] https://github.com/mqtt-tools/pytest-mqtt/blob/0.1.0/pytest_mqtt/mosquitto.py#L44-L54